cypress/integration/homeview_spec.js

```diff
@@ -1,14 +1,9 @@
 describe('VegShare, main page view', () => {
-  const baseUrl = 'https://veg-share.herokuapp.com'
   beforeEach(() => {
-    cy.visit(baseUrl)
+    cy.visit('/')
   })
-  it ('should have the correct url for the home page on load', () => {
-    cy
-      .url().should('eq', `${baseUrl}/`)
-  });
   it ('should show the site header', () => {
     cy
-      .get('header .veg-share-title').should('contain', 'Veg-Share')
+      .get('header .veg-share-title').should('contain', 'Garden Party')
```

Comment:
Hmm I can't seem to reply to your other comment, but that's not the deployed site, that's the home page for the Vercel site for our app. I might have to add you two or something?

This is the deployed site - https://frontend-ui.vercel.app/ - without the updated changed of course because they haven't been merged to main.